plugin/storage/grpc/proto/storage.proto

```diff
@@ -66,16 +66,20 @@ message GetTraceRequest {
       (gogoproto.customtype) = "github.com/jaegertracing/jaeger/model.TraceID",
       (gogoproto.customname) = "TraceID"
     ];
+    string bearerToken = 2;
 }
 
-message GetServicesRequest {}
+message GetServicesRequest {
+    string bearerToken = 1;
```

Comment:
I don't think it's a good idea to explicitly change the storage API for this cross-cutting concern. gRPC requests can pass metadata similar to HTTP headers, I think it would be a much better place for auth tokens.

Reply:
Indeed, updated code does not do this anymore.